test/unit/datasets/DatasetsRepository.test.ts

```diff
@@ -141,23 +135,23 @@ describe('DatasetsRepository', () => {
 
       assert.calledWithExactly(
         axiosGetStub,
-        `${testApiUrl}/datasets/${testDatasetModel.id}/versions/${testDatasetModel.versionId}`,
-        {
-          withCredentials: true,
-        },
+        `${TestConstants.TEST_API_URL}/datasets/${testDatasetModel.id}/versions/${testDatasetModel.versionId}`,
+        TestConstants.TEST_EXPECTED_AUTHENTICATED_REQUEST_CONFIG,
       );
```

Comment:
Why do you remove the withCredentials test? If I understood the code, we still support withCredentials, right?